example/main.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+import time
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+import eppo_client  # noqa
+from eppo_client.config import Config  # noqa
+from eppo_client.assignment_logger import AssignmentLogger  # noqa
+
+API_KEY = "REPLACE WITH YOUR API KEY"
+
+
+class ExampleAssignmentLogger(AssignmentLogger):
+    def log_assignment(self, assignment):
+        print(
+            f'{assignment["subject"]} assigned {assignment["variation"]} for key {assignment["experiment"]}'
+        )
+
+
+def init_eppo_and_assign():
+    client_config = Config(api_key=API_KEY, assignment_logger=ExampleAssignmentLogger())
+    eppo_client.init(client_config)
+
+    eppo = eppo_client.get_instance()
+
+    # ensure the client is initialized before assigning
+    time.sleep(1)
```

Comment:
@schmit whats the canonical way in python to be able to avoid this? https://docs.launchdarkly.com/sdk/server-side/python

Reply:
Not sure, no expert on threads -- I think normally this isn't really an issue for the server side to wait, just this example